test/e2e/log.go

```diff
@@ -0,0 +1,35 @@
+package e2e
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/onsi/ginkgo"
+)
+
+const (
```

Comment:
I agree that this isn't that nice, the thing that I wanted to use the GinkgoWriter, because other logs don't work well with Ginkgo tests (IIRC you only see other logs for failed tests).

Reply:
Can we at least add some comments to this file to explain the need?

Reply:
done